packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -269,41 +269,24 @@ class RenderAndroidView extends PlatformViewRenderBox {
   }
 }
 
-/// A render object for an iOS UIKit UIView.
-///
-/// [RenderUiKitView] is responsible for sizing and displaying an iOS
-/// [UIView](https://developer.apple.com/documentation/uikit/uiview).
-///
-/// UIViews are added as sub views of the FlutterView and are composited by Quartz.
-///
-/// {@macro flutter.rendering.RenderAndroidView.layout}
-///
-/// {@macro flutter.rendering.RenderAndroidView.gestures}
-///
-/// See also:
-///
-///  * [UiKitView] which is a widget that is used to show a UIView.
-///  * [PlatformViewsService] which is a service for controlling platform views.
-class RenderUiKitView extends RenderBox {
-  /// Creates a render object for an iOS UIView.
+/// Superclass for common fucntionality of platform view RenderBoxes
+class RenderDarwinPlatformView<T extends DarwinPlatformViewController> extends RenderBox {
+  /// Creates a render object for a platform view.
   ///
   /// The `viewId`, `hitTestBehavior`, and `gestureRecognizers` parameters must not be null.
```

Comment:
You could also just remove this sentence entirely since we no longer document that something must not be null now that we have full null safety and the types are self-documenting.